fennel/connectors/connectors.py

```diff
@@ -106,6 +117,7 @@ def source(
     bounded: bool = False,
     idleness: Optional[Duration] = None,
     where: Optional[Callable] = None,
+    sample: Optional[Union[float, Sample]] = None,
```

Comment:
Why not Optional[Sample]?

Reply:
User can give float directly without specifying any columns
https://coda.io/d/Sampled-Ingestion_dkplIi5d_Kp/Sampled-Ingestion_suIbBFju#_lu9Aph4j